assets/src/dashboard/app/views/templates/components.js

```diff
@@ -26,6 +26,7 @@ import { Button } from '../../../components';
 
 export const ContentContainer = styled.div`
   ${({ theme }) => `
+    margin-top: ${theme.navBar.height + 40}px;
     padding: 0 ${theme.pageGutter.large.desktop}px;
```

Comment:
nit: I would recommend switching the padding and margin around. This is primarily because margin gets collapsed on the parent and behaves funky for vertical stuff, whereas padding doesn't

 it's also nice to have margin horizontally in case we ever want to utilize a `margin: 0 auto` on larger viewports.

I think as a general rule of thumb, padding behave nicer for vertical positioning whereas margin is nicer for horizontal

Reply:
Sounds good, I've never had margins collapse like that but this reasoning is really straight forward and makes sense.